packages/itwinui-css/backstop/tests/tabs.html

```diff
@@ -46,85 +46,69 @@ <h2>Default</h2>
-              aria-controls="disabled-tab"
-              disabled
-            >
-              <svg-placeholder
-                class="iui-tab-icon"
-                aria-hidden="true"
-              ></svg-placeholder>
-              <span class="iui-tab-label">
-                <div>Disabled</div>
-                <div class="iui-tab-description">Sublabel</div>
-              </span>
-            </button>
-          </li>
-          <li>
-            <button
-              class="iui-tab"
-              role="tab"
-              aria-selected="false"
-              aria-controls="none-tab"
-            >
-              <span class="iui-tab-label">None</span>
-            </button>
-          </li>
-          <li>
-            <button
-              class="iui-button"
-              data-iui-variant="borderless"
-            >
-              <svg-add-circular
-                class="iui-button-icon"
-                aria-hidden="true"
-              ></svg-add-circular>
-            </button>
-          </li>
-        </ul>
+          <button
+            class="iui-tab iui-active"
+            role="tab"
+            aria-selected="true"
+            aria-controls="active-tab"
+          >
+            <svg-placeholder
+              class="iui-tab-icon"
+              aria-hidden="true"
+            ></svg-placeholder>
+            <span class="iui-tab-label">Active</span>
+            <span class="iui-tab-description">Sublabel</span>
+          </button>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="left-tab"
+          >
+            <svg-placeholder
+              class="iui-tab-icon"
+              aria-hidden="true"
+            ></svg-placeholder>
+            <span class="iui-tab-label">Left</span>
+            <span class="iui-tab-description">Sublabel</span>
+          </button>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="disabled-tab"
+            disabled
+          >
+            <svg-placeholder
+              class="iui-tab-icon"
+              aria-hidden="true"
+            ></svg-placeholder>
+            <span class="iui-tab-label">Disabled</span>
+            <span class="iui-tab-description">Sublabel</span>
+          </button>
+          <button
+            class="iui-tab"
+            role="tab"
+            aria-selected="false"
+            aria-controls="none-tab"
+          >
+            <span class="iui-tab-label">None</span>
+          </button>
+          <button
+            class="iui-button"
+            data-iui-variant="borderless"
+          >
+            <svg-add-circular
+              class="iui-button-icon"
+              aria-hidden="true"
+            ></svg-add-circular>
+          </button>
+        </div>
 
         <div class="iui-tabs-actions-wrapper">
           <div class="iui-tabs-actions">
```

Comment:
I want to open a discussion about `iui-tabs-actions-wrapper` and `iui-tabs-actions`.

In all of these HTML examples and in the old React API for Tabs, `iui-tabs-actions-wrapper` only has one child which is `iui-tabs-actions`. When you place multiple `iui-tabs-actions` elements in the wrapper, this is the result for a horizontal and vertical tab

![image](https://github.com/iTwin/iTwinUI/assets/78564221/2828a21a-b541-4588-a38c-75ee384176e8)
![image](https://github.com/iTwin/iTwinUI/assets/78564221/1db28598-2fbf-42b1-9531-f4cfd3985b56)

If the horizontal example is intended where adding a new `iui-tabs-actions` element creates a new row of action buttons, then this needs to be true for creating a new column of action buttons in the vertical example and the spacing needs to be fixed

If this isn't intended and the wrapper is only supposed to wrap around a single `actions` element, can we look into removing `iui-tabs-actions-wrapper` altogether?

Reply:
i believe the wrapper was needed for grid placement. it might not be possible to remove the class, but you can remove the react component and add `wrapperProps` to `Tabs.Actions`.